.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,152 @@
+name: Emscripten
+
+on: [push, pull_request]
+
+jobs:
+  build-win10:
+    strategy:
+      matrix:
+        config: [Debug, Release]
+
+    runs-on: windows-latest
+    name: Win10 -> Emscripten, ${{ matrix.config }}
+
+    steps:
+    - name: Clone DiligentTools
+      uses: actions/checkout@v2
+      with:
+        path:       DiligentTools
+        submodules: recursive
+
+    - name: Clone DiligentCore
+      uses: actions/checkout@v2
+      with:
+        repository: 'DiligentGraphics/DiligentCore'
+        path:       DiligentCore
+        submodules: recursive
+
+    - name: Configure Ninja
+      uses: seanmiddleditch/gha-setup-ninja@master
+
+    - name: Configure Emscripten SDK version
+      if: success()
+      shell: bash
+      run: |
+        cd $GITHUB_WORKSPACE/DiligentCore/BuildTools/Scripts/github_actions
+        ./emscripten_sdk.sh "$GITHUB_WORKSPACE"
+
+    - name: Configure CMake
+      if: success()
+      shell: bash
+      run: |
+        cd $GITHUB_WORKSPACE/DiligentTools/BuildTools/Scripts/github_actions
+        ./create_cmake.sh "$GITHUB_WORKSPACE"
+        cd $GITHUB_WORKSPACE/DiligentCore/BuildTools/Scripts/github_actions
+        ./configure_cmake.sh "emscripten" "${{runner.workspace}}" ${{ matrix.config }} "-DDILIGENT_BUILD_TOOLS_INCLUDE_TEST=ON" "-DDILIGENT_BUILD_TOOLS_TESTS=ON"
```

Comment:
Should be "-DDILIGENT_BUILD_TOOLS_INCLUDE_TEST=ON -DDILIGENT_BUILD_TOOLS_TESTS=ON" - one string parameter. Otherwise the last parameter will be ignored

Reply:
Yes this is a typo